hyperspy/docstrings/model.py

```diff
@@ -0,0 +1,99 @@
+                  Levenberg-Marquardt algorithm, and supports bounds
+                  on parameters.
+                * "trf" performs least-squares optimization using the
+                  Trust Region Reflective algorithm, and supports
+                  bounds on parameters.
+                * "dogbox" performs least-squares optimization using the
+                  dogleg algorithm with rectangular trust regions, and
+                  supports bounds on parameters.
+                * "odr" performs the optimization using the orthogonal
+                  distance regression (ODR) algorithm. It does not support
+                  bounds on parameters. See :py:mod:`scipy.odr` for more details.
+                * All of the available methods for :py:func:`scipy.optimize.minimize`
+                  can be used here. See the :ref:`User Guide <model.fitting>`
+                  documentation for more details.
+                * "Differential Evolution" is a global optimization method.
+                  It does support bounds on parameters. See
+                  :py:func:`scipy.optimize.differential_evolution` for more
+                  details on available options.
+                * "Dual Annealing" is a global optimization method.
+                  It does support bounds on parameters. See
+                  :py:func:`scipy.optimize.dual_annealing` for more
+                  details on available options. Requires ``scipy >= 1.2.0``.
+                * "SHGO" (simplicial homology global optimization" is a global
+                  optimization method. It does support bounds on parameters. See
+                  :py:func:`scipy.optimize.shgo` for more details on available
+                  options. Requires ``scipy >= 1.2.0``.
+
+        loss_function : {"ls", "ml-poisson", "huber", callable}, default "ls"
+            The loss function to use for minimization. Only ``"ls"`` is available
+            if ``optimizer`` is one of ``["lm", "trf", "dogbox", "odr"]``.
+
+                * "ls" minimizes the least-squares loss function.
+                * "ml-poisson" minimizes the negative log-likelihood for
+                  Poisson-distributed data. Also known as Poisson maximum
+                  likelihood estimation (MLE).
+                * "huber" minimize the Huber loss function. The delta value
+                  of the Huber function is controlled by the ``huber_delta``
+                  keyword argument (the default value is 1.0).
+                * callable supports passing your own minimization function.
+
+        grad : {"auto", "analytical", callable, None}, default "auto"
+            Whether to use information about the gradient of the loss function
+            as part of the optimization. This parameter has no effect if
+            ``optimizer`` is a derivative-free or global optimization method.
+
+                * "auto" uses a finite difference scheme (if available) for numerical
```

Comment:
I think that it'll be clearer to rename `"auto"` -> `"fd"`

Reply:
Done